.github/workflows/end2end-test.yml

```diff
@@ -60,6 +60,97 @@ jobs:
               uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb # v3.1.1
               if: always()
               with:
-                  name: flaky-tests-report-${{ matrix.part }}
+                  name: flaky-tests-report
                   path: flaky-tests
                   if-no-files-found: ignore
+
+    e2e-playwright:
```

Comment:
Agreed. I didn't change it simply because I don't know if any other code depends on the names. Probably we would need to update the required checks settings but we already have to do that for playwright tests anyway.